run_desktop_tests.sh

```diff
@@ -40,6 +40,16 @@ cargo check --release --target=thumbv7em-none-eabi --examples
 echo "Checking that CTAP2 builds and links properly (1 set of features)..."
 cargo build --release --target=thumbv7em-none-eabi --features with_ctap1
 
+echo "Checking that supported boards build properly..."
+cd third_party/tock/boards
+cd nordic/nrf52840dk
+make
+cd ../..
+cd nordic/nrf52840_dongle
+make
```

Comment:
We can as well do `make -C third_party/tock/boards/nordic/nrf52840dk`.

Reply:
+1 on this modification